packages/taquito-utils/src/taquito-utils.ts

```diff
@@ -86,6 +86,7 @@ export function b58decode(payload: string) {
     [prefix.tz1.toString()]: '0000',
     [prefix.tz2.toString()]: '0001',
     [prefix.tz3.toString()]: '0002',
+    [prefix.tz4.toString()]: '0003',
```

Comment:
Curious to know if the 0003 encodings come from the RPC?
The bytes version of tz4VHgLiRx5ZZjwU2QaybHc11EMJk3NcyvVc gives me de7dd60e638ee1b5b5e520b2a5fcaf91af76bc3f which doesn't start with 0003

Reply:
If you query the contract deployed in your integration (the one having tx_rollup_l2_address in storage) with different RPC endpoint, you can see both format in the storage value.
String: https://jakartanet.ecadinfra.com/chains/main/blocks/head/context/contracts/KT1BxzSKyeXSxnBfjHf3597oED1kTuXLtUM6/
Bytes:https://jakartanet.ecadinfra.com/chains/main/blocks/head/context/contracts/KT1BxzSKyeXSxnBfjHf3597oED1kTuXLtUM6/script